### PR TITLE
Update docs after playwright switch

### PR DESCRIPTION
**Pull request by boubkerbribri**
Update docs after playwright switch (merge after https://github.com/PrestaShop/PrestaShop/pull/19653)

### CODE REVIEW

File: src/content/1.7/testing/ui-tests/how-to-contribute-and-create-web-acceptance-tests.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 ---
-title: How to contribute and create web acceptance tests
-menuTitle: Creation of web acceptance tests
+title: How to contribute and create UI tests
+menuTitle: Creation of UI tests
 weight: 2
 aliases:
-   - /1.7/testing/how-to-create-your-own-web-acceptance-tests
+   - /1.7/testing/how-to-create-your-own-ui-tests
```
Comment (Progi1984) [inline src/content/1.7/testing/ui-tests/how-to-contribute-and-create-web-acceptance-tests.md:6]:
```suggestion
   - /1.7/testing/how-to-create-your-own-web-acceptance-tests
```

Comment (SimonGrn) [reply src/content/1.7/testing/ui-tests/how-to-contribute-and-create-web-acceptance-tests.md:6]: Can't we have some sort of redirection ?

Comment (Progi1984) [reply src/content/1.7/testing/ui-tests/how-to-contribute-and-create-web-acceptance-tests.md:6]: If I remember, redirection is for pages which have already created and referenced by BigG. 
Aliases are redirections.

Comment (PierreRambaud) [reply src/content/1.7/testing/ui-tests/how-to-contribute-and-create-web-acceptance-tests.md:6]: Exactly, aliases are the "old urls"  https://gohugo.io/content-management/urls/

File: src/content/1.7/testing/ui-tests/_index.md [hunk 2]
```diff
@@ -1,21 +1,21 @@
 ---
-title: Web acceptance tests
+title: UI tests
 weight: 30
 chapter: true
```
Comment (Progi1984) [inline src/content/1.7/testing/ui-tests/_index.md:4]:
```suggestion
chapter: true
aliases:
  - /1.7/testing/web-acceptance-tests
```

File: src/content/1.7/testing/ui-tests/how-to-contribute-and-create-ui-tests.md [hunk 3]
```diff
@@ -1,12 +1,13 @@
 ---
-title: How to contribute and create web acceptance tests
-menuTitle: Creation of web acceptance tests
+title: How to contribute and create UI tests
+menuTitle: Creation of UI tests
 weight: 2
 aliases:
    - /1.7/testing/how-to-create-your-own-web-acceptance-tests
+   - /1.7/testing/web-acceptance-tests
```
Comment (Progi1984) [inline src/content/1.7/testing/ui-tests/how-to-contribute-and-create-ui-tests.md:7]:
```suggestion
   - /1.7/testing/web-acceptance-tests/how-to-contribute-and-create-web-acceptance-tests/
```

**Comment by Progi1984** [comment]
Thanks @boubkerbribri

**Pull request merged**